spec/account_spec.rb

```diff
@@ -0,0 +1,98 @@
+require "spec_helper"
+require "bankaccount"
+
+describe BankAccount::Account do
+	
+	it "responds to class method called new" do
+		expect(BankAccount::Account).to respond_to :new
+	end
+
+	describe "it creates an instance of a new Account with an id and a positive initial_balance" do
```

Comment:
When you run your specs, errors will be displayed as the `describe` strings followed by the `it` string.
As such the `describe` strings should be short and descriptive of all the specs nested within it.
One example that might work here is "Initialized account"
Then everything in that section should test that the initialized account and the initialization process is as you think it should be.
`it` strings should let you know what the problem is before you go and look at the code.
What we are testing here is that it sets things if it is valid and raising errors if invalid.
As such your 3rd test in this describe block has a very good description.
The other two are testing the general concept of setting the id and the initial balance. The exact value is part of the test itself but not really the display string.